### DISCUSSION TITLE
getPut() fetch incorrect data

### DISCUSSION THREAD

**Discussion by diegohdez90** [phalcon/cphalcon#15575] [Q&A]
I was writing a code to get PUT body request data. But, it returns incorrect data from body data

![image](https://user-images.githubusercontent.com/10326623/124319328-a9048380-db3f-11eb-9cd7-08b111620244.png)

If you notice, the `email` value send me with underscore value from `getPUT()` method.

```php
/** getPUT() */
$request = new Request();
$data = $request->getPut();

...
```

This is the way I use `getPUT()`

Phalcon version: 3.4.5
PHP version: 7.2.24 (I know, this is deprecated)
Operating System: Linux.

**Comment by Jeckerson** [comment]
Hello, are you sure that you do not have any email filters enabled? Also Phalcon `3.4.5` is really old version.

**Reply by diegohdez90** [reply]
Yeah, in my job already are afraid to update. Also, when I was assigned in the project. I read the code and there is not migration file, all database was created in a hardcoded way.

Thanks for your answer @Jeckerson, I'll review if exists a email filter. Do you know in what directory exists?
